TLM/TLM/Lifecycle/SerializableDataExtension.cs

```diff
@@ -153,11 +181,113 @@ public class SerializableDataExtension
             }
         }
 
+        private static void LoadDomCollection() {
```

Comment:
"Hard to read for human" is relative. It's certainly more readable than the binary format we were using before. Its readability has already been a valuable tool for me in working on #1392.

On your other points, the original choice was Json, but we were unable to find a trustworthy and full-featured Json serializer that was compatible with the particular combination of Unity and Mono versions we have to work with. Json.net is not compatible, and the Unity port of Json.net is abandoned and also nonfunctional.

The main advantage of XML is that we can use a known format and proven, stable code. If we rolled our own, we would be taking on a whole new level of maintenance responsibility.

One way we could reduce size is by using very short names, but this would pretty much destroy readability.

I'm open to other suggestions.......